nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -78,15 +78,15 @@ config PSA_CRYPTO_DRIVER_ALG_GCM_OBERON
 
 config PSA_CRYPTO_DRIVER_ALG_HKDF_OBERON
 	bool
-	prompt "PSA HKDF support - oberon"
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_HKDF && \
-		     !PSA_CRYPTO_DRIVER_ALG_HKDF_CC3XX
+	#prompt "PSA HKDF support - oberon" - Currently not supported
+	#default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_HKDF && \
+		     #!PSA_CRYPTO_DRIVER_ALG_HKDF_CC3XX
```

Comment:
If not supported, then why not remove this Kconfig.
The commit can always be reverted, when support has been added.

Reply:
We are keeping it in until we have qualified that the missing API is the reason for test-failures. This will be cleaned up before this goes in (as well as other comments with # in front)

Reply:
could then please turn this PR into draft mode, so we know when to review ?

If this is not in draft mode, to allow CI to run, then at least indicate that this is not ready for review in the title and / or PR description

Reply:
Kconfigs that are no longer available have been removed

Reply:
@tejlmand : I believe this is ready for review now